doc/developer-guides/hld/hv-hypercall.rst

```diff
@@ -9,6 +9,28 @@ is from the hypervisor to the guest VM. The hypervisor supports
 hypercall APIs for VM management, I/O request distribution, interrupt injection,
 PCI assignment, guest memory mapping, power management, and secure world switch.
 
+The application binary interface (ABI) of ACRN hypercalls is defined as follows.
+
+- A guest VM executes the ``vmcall`` instruction to trigger a hypercall.
+
+- Input parameters of a hypercall includes:
+
+  - An hypercall ID in ``R8``, which specifies the kind of service requested by
+    the guest VM.
+
+  - The first parameter in ``RDI`` and the second in ``RSI``. The semantics of
+    those two parameters vary among different kinds of hypercalls and are
+    defined in the hypercall APIs reference. For hypercalls targeting a specific
+    VM, the first parameter is typically the ID of that VM.
```

Comment:
Why would a hypercall target a specific VM? The introduction above says a hypercall is from a VM to the hypervisor.

Reply:
Using ``target`` here could be ambiguous. I rephrased it to ``For hypercalls requesting operations on a specific VM``.